src/neptune/new/internal/disk_queue.py

```diff
@@ -170,6 +177,22 @@ def close(self):
         self._last_ack_file.close()
         self._last_put_file.close()
 
+        if self.is_empty():
+            self._remove_data()
+
+    def _remove_data(self):
+        path = self._dir_path
+        shutil.rmtree(path, ignore_errors=True)
+
+        parent = path.parent
+
+        files = os.listdir(parent)
+        if len(files) == 0:
+            try:
+                os.rmdir(parent)
+            except OSError:
+                _logger.info(f"Cannot remove directory: {parent}")
```

Comment:
Looks like something from another MR/task. Probably something went wrong during a rebase?

Reply:
Fixed, for some reason those changes weren't visible to me locally